updated GitBook to Docusaurus

DougVaderJr opened the pull request:
Updates: replace GitBook OSS docs with Docusaurus platform

Commit:
updated GitBook to Docusaurus

Updates GitBook to Docusaurus

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/electrode-io/electrode?pullRequest=1673) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/electrode-io/electrode?pullRequest=1673) before we can accept your contribution.<br/><hr/>**Rob** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/electrode-io/electrode?pullRequest=1673) it.</sub>

jchip commented on README.md:
````diff
@@ -4,65 +4,34 @@
 
-- [Getting Started with Electrode]
+```bash
+npm install -g xclap-cli
````
these instructions don't work.

```
mkdir my-app
cd my-app
npm init @xarc/app
npm install
npm run dev
```

Commit:
Update README.md

datvong-wm commented on docs/docusaurus.config.js:
```diff
@@ -0,0 +1,90 @@
+          sidebarPath: require.resolve('./sidebars.js'),
+          // Please change this to your repo.
+          editUrl:
+            'https://gecgithub01.walmart.com/electrode/docs',
```
internal

datvong-wm commented on docs/docusaurus.config.js:
```diff
@@ -0,0 +1,90 @@
+          showReadingTime: true,
+          // Please change this to your repo.
+          editUrl:
+            'https://gecgithub01.walmart.com/electrode/docs',
```
update url

jchip commented on README.md:
```diff
@@ -4,65 +4,36 @@
 
-![Hello from Electrode][hello-electrode]
+If you are a Walmart developer, please see the internal electrode X documentation site.
+:::
```
We maintain additional code to support internal systems.  If you are a Walmart developer, please check internal docs to get started.

jchip commented on README.md:
```diff
@@ -4,65 +4,36 @@
-
-Check [gitbook] for more info about gitbook.
-
-# Contributing
```
we should keep contributing section

jchip commented on README.md:
```diff
@@ -4,65 +4,36 @@
 
-**It takes 5 minutes and a single command to get your own Universal React app to customize:**
+:::warning
+This framework is for the open-source version of electrode.
```
this makes it sound like we have two versions.  we don't.

Commit:
Add footer info

ADD: 'Walmart Labs' to footer in Subapp demo

Commit:
Merge branch 'master' of https://github.com/DougVaderJr/electrode

Commit:
Change footers in demo app

CHANGE: "Your (Company) Name Here" to "Walmart Labs" in footer of `Subapp-demo2.js`

Commit:
Delete OSS version warning

remove "This is for the OSS version of electrode."

Commit:
Final edits

Commit edits from GTP.js team

Commit:
Merge branch 'master' of https://github.com/DougVaderJr/electrode

Commit:
Final minor edits

Change Quick Start console app scaffold commands
Remove/replace footer info "Your co. name here"

Commit:
Update image

updated image removing WMT footer

Commit:
Update README.md

jchip commented on README.md:
```diff
@@ -4,65 +4,40 @@
+# Building large scale universal React Web Applications
 
-### Building large scale universal React Web Applications
+Electrode Web is a rapid application development framework; a full stack, end-to-end platform for developing, deploying, and maintaining JavaScript applications at Walmart, afor the <http://www.walmart.com> eCommerce website.
```
`afor`?

jchip commented on README.md:
```diff
@@ -4,65 +4,40 @@
+Electrode Web is a rapid application development framework; a full stack, end-to-end platform for developing, deploying, and maintaining JavaScript applications at Walmart, afor the <http://www.walmart.com> eCommerce website.
 
-> Built at @WalmartLabs powering the <http://www.walmart.com> eCommerce site, Electrode is a platform for building small to large scale universal ReactJS/NodeJS web applications with a standardized structure, best practices, and modern technologies baked in. Electrode focuses on performance, component reusability, and simple deployment to multiple cloud providers—so you can focus on what makes yo ...
+We maintain additional code to support internal systems.
```
For Walmart applications, we maintain additional code to support proprietary internal systems.  If you are a Walmart developer working on internal applications, please check internal docs to get started.

Commit:
updates

update readme.md

Commit:
update readme

Image in readme.md
